autoload/ale/list.vim

```diff
@@ -103,6 +103,9 @@ function! s:SetListsImpl(timer_id, buffer, loclist) abort
         endfor
     endif
 
+    " TODO: Can either be a buffer variable or go into g:ale_buffer_info?
+    let b:ale_winview = winsaveview()
```

Comment:
I think this is good a buffer variable. Use `setbufvar` with `a:buffer`, which could be a different buffer from the current noe.

Reply:
Done!